symfony/client/src/orangehrmAuthenticationPlugin/pages/AdministratorAccess.vue

```diff
@@ -0,0 +1,184 @@
+<!--
+/**
+ * OrangeHRM is a comprehensive Human Resource Management (HRM) System that captures
+ * all the essential functionalities required for any enterprise.
+ * Copyright (C) 2006 OrangeHRM Inc., http://www.orangehrm.com
+ *
+ * OrangeHRM is free software; you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with this program;
+ * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA  02110-1301, USA
+ */
+ -->
+
+<template>
+  <div class="orangehrm-admin-access-container">
+    <div class="orangehrm-card-container">
+      <oxd-form
+        ref="verifyForm"
+        method="post"
+        :action="submitUrl"
+        @submitValid="onSubmit"
+      >
+        <oxd-text class="orangehrm-admin-access-title">
+          Administrator Access
+        </oxd-text>
+
+        <br />
+
+        <oxd-text class="orangehrm-admin-access-note">
+          You have requested to access a critical Administrator function in
+          OrangeHRM and are required to validate your credentials below
+        </oxd-text>
+
+        <br v-if="error === null" />
```

Comment:
```
<oxd-alert v-if="error !== null" type="error" :message="error?.message"/>
<br v-else />
```

Reply:
can't change it like this since `oxd-alert` requires the `show` prop in order to display.

To use v-else can write like this

```
<oxd-alert
          v-if="error !== null"
          :show="error !== null"
          type="error"
          :message="error?.message"
        />
<br v-else />
```

Reply:
I don't think it's good. looks like `oxd-alert` is having another `v-if` inside the component. in this case we will have to keep it like before or we can remove `<br>` can't we achieve the spacing via css padding ?

Reply:
okay I'll try to do via padding

Reply:
Hi @Super-Chama is the way I've done it okay?